app/components/zendesk-request-form.js

```diff
@@ -0,0 +1,108 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import { reads, bool } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import $ from 'jquery';
+import moment from 'moment';
+import config from 'travis/config/environment';
+
+const { apiHost, createRequestEndpoint } = config.zendesk;
+
+const UTC_START_TIME = moment.utc({ h: 9, m: 0, s: 0 });
+const UTC_END_TIME = moment.utc({ h: 23, m: 0, s: 0 });
+const DATE_FORMAT = 'LT';
+
+export default Component.extend({
+  classNames: ['zendesk-request-form'],
+
+  auth: service(),
+  flashes: service(),
+
+  page: '',
+
+  email: reads('auth.currentUser.email'),
+
+  subject: '',
+
+  description: reads('descriptionTemplate'),
+
+  descriptionTemplate: computed(function () {
+    return buildDescriptionTemplate(this.page);
+  }),
+
+  isLoggedIn: reads('auth.signedIn'),
+
+  startTime: UTC_START_TIME.local().format(DATE_FORMAT),
+  endTime: UTC_END_TIME.local().format(DATE_FORMAT),
+  timezone: moment.tz(moment.tz.guess()).format('z'),
+
+  isSubmitting: reads('zendeskRequest.isRunning'),
+  isSuccess: bool('zendeskRequest.lastSuccessful.value'),
+
+  zendeskRequest: task(function* () {
+    const { fullName: name, email } = this.auth.currentUser;
+    const { subject, description: body } = this;
+
+    try {
+      return yield $.post(`${apiHost}/${createRequestEndpoint}`, {
+        request: {
+          requester: { name, email },
+          subject,
+          comment: { body }
+        }
```

Comment:
I haven’t been able to log in to Zendesk but the autoreply I received showed this as a subject:

![image](https://user-images.githubusercontent.com/43280/49249196-3fe58e00-f3e1-11e8-8fce-4382be28a02b.png)

That stray `}` is curious, should this maybe be `comment: body`? Just a guess.

Reply:
I was able to log in finally and the `}` isn’t there. So I’m not sure what the origin is. Maybe it’s not a problem with this code!

Reply:
That's interesting! Thanks for pointing, I'll test more around this issue. But I don't think it's relevant to this code, because according to [Zendesk API](https://developer.zendesk.com/rest_api/docs/support/requests#create-request) `body` is nested inside `comment` 🤔